shared/js/media/video_player.js

```diff
@@ -471,6 +483,20 @@ function VideoPlayer(container) {
     }
   });
 
+  slider.addEventListener('keypress', function(e) {
+    // Currently, we only allow screen reader users to adjust sliders with a
+    // constant step size (there is no page up/down equivalent). In the case
+    // of videos, we make sure that the maximum amount of steps for the entire
+    // duration is 20, or 2 second increments if the duration is less then 40
+    // seconds.
+    var step = Math.max(player.duration/20, 2);
+    if (e.keyCode == e.DOM_VK_DOWN) {
+      player.currentTime -= step;
+    } else {
```

Comment:
The screen reader is currently broken in gecko-37 and m-c. A patch in [bug 1125422](https://bugzilla.mozilla.org/show_bug.cgi?id=1125422) is about to land in inbound once the tree opens (and should eventually be uplifted to 37).

But generally, swipe up/down is already implemented and works great. I tested this locally with the patches from the bug above applied.